ui/app/src/services/content.ts

```diff
@@ -17,7 +17,13 @@
 import { get, getText, post, postJSON } from '../utils/ajax';
```

Comment:
Please revert the **format** changes of this file so that I can review *your* changes.